features/gezag-minderjarige/1.2-minderjarig-niet-overleden.feature

```diff
@@ -0,0 +1,296 @@
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                          |
+      | type        | GezagNietTeBepalen                                                                              |
+      | toelichting | gezag is niet te bepalen omdat de volgende relevante gegevens in onderzoek staan: <toelichting> |
+
+      Voorbeelden:
+      | meer of minderjarig | aanduiding in onderzoek | onderzoek omschrijving | toelichting                                            |
+      | minderjarig         | 010000                  | hele categorie persoon | geboortedatum van de persoon, geboorteland van persoon |
+      | minderjarig         | 010300                  | hele groep geboorte    | geboortedatum van de persoon, geboorteland van persoon |
+      | minderjarig         | 010310                  | geboortedatum          | geboortedatum van de persoon                           |
+      | meerderjarig        | 010310                  | geboortedatum          | geboortedatum van de persoon                           |
+
+    Abstract Scenario: bij minderjarige staat een ander gegeven dan geboortedatum, namelijk <onderzoek omschrijving> in onderzoek
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft de volgende onderzoek gegevens
+      | aanduiding in onderzoek (83.10) |
+      | <aanduiding in onderzoek>       |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+      Voorbeelden:
+      | aanduiding in onderzoek | onderzoek omschrijving |
+      | 010120                  | burgerservicenummer    |
+      | 010200                  | hele groep naam        |
+      | 010240                  | geslachtsnaam          |
+      | 010320                  | geboorteplaats         |
```

Comment:
geboorteplaats wordt wel gebruikt voor gezagsbepaling toch? Moet dit in onderzoek dan niet leiden tot een niet te bepalen?

Reply:
Geboorteplaats wordt niet gebruikt in de gezagsmodule. Waarschijnlijk denk je aan geboorteland.